CHANGES.txt

```diff
@@ -5,6 +5,9 @@ CHANGES for crate-dbal
 Unreleased
 ==========
 
+ - Fixed an issue that ignores provided ``UNIQUE`` constraint. The client now
```

Comment:
I would say "Fixed an issue that ignored provided...." and "throws and Exception and informs the user to use the"